editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -1925,16 +1949,17 @@ void VisualShaderEditor::_draw_color_over_button(Object *obj, Color p_color) {
 	button->draw_rect(Rect2(normal->get_offset(), button->get_size() - normal->get_minimum_size()), p_color);
 }
 
-void VisualShaderEditor::_update_created_node(GraphNode *node) {
-	const Ref<StyleBoxFlat> sb = node->get_theme_stylebox(SNAME("frame"), SNAME("GraphNode"));
-	Color c = sb->get_border_color();
+//TODO @Geometror: Solve this
```

Comment:
Solve what? :)

Reply:
Mmh, I solved it by removing the comment :)

I think this was a relict of an early version of this rework.